admin/tools/qserv-install.sh

```diff
@@ -7,8 +7,8 @@ DIR=$(cd "$(dirname "$0")"; pwd -P)
 
 # Default values below may be overidden by cmd-line options
 MODE="internet mode"
-NEWINSTALL_URL="https://raw.githubusercontent.com/lsst/lsst/master/scripts/newinstall.sh"
-VERSION="-t qserv"
+NEWINSTALL_URL="https://raw.githubusercontent.com/lsst/lsst/12.0/scripts/newinstall.sh"
```

Comment:
LSST documentation recommend using the release link:
https://pipelines.lsst.io/install/newinstall.html
master could be broken as it is development version, isn't it?